code/datums/brain_damage/hypnosis.dm

```diff
@@ -25,14 +25,14 @@
 /datum/brain_trauma/hypnosis/on_gain()
 	message_admins("[ADMIN_LOOKUPFLW(owner)] was hypnotized with the phrase '[hypnotic_phrase]'.")
 	owner.log_message("was hypnotized with the phrase '[hypnotic_phrase]'.", LOG_GAME)
-	to_chat(owner, "<span class='reallybig hypnophrase'>[hypnotic_phrase]</span>")
-	to_chat(owner, "<span class='notice'>[pick("You feel your thoughts focusing on this phrase... you can't seem to get it out of your head.",\
+	to_chat(owner, span_reallybig(span_hypnophrase("[hypnotic_phrase]")))
+	to_chat(owner, span_notice("[pick("You feel your thoughts focusing on this phrase... you can't seem to get it out of your head.",\
 												"Your head hurts, but this is all you can think of. It must be vitally important.",\
 												"You feel a part of your mind repeating this over and over. You need to follow these words.",\
 												"Something about this sounds... right, for some reason. You feel like you should follow these words.",\
-												"These words keep echoing in your mind. You find yourself completely fascinated by them.")]</span>")
-	to_chat(owner, "<span class='boldwarning'>You've been hypnotized by this sentence. You must follow these words. If it isn't a clear order, you can freely interpret how to do so,\
-										as long as you act like the words are your highest priority.</span>")
+												"These words keep echoing in your mind. You find yourself completely fascinated by them.")]"))
+	to_chat(owner, span_boldwarning("You've been hypnotized by this sentence. You must follow these words. If it isn't a clear order, you can freely interpret how to do so, \
+										as long as you act like the words are your highest priority."))
```

Comment:
I think you can change this to fit all one line? might be long though.

Reply:
cant really make it fit in one line easily, i divided it in a way that at least is more up to date